api/db_queries.py

```diff
@@ -37,3 +37,31 @@
             RETURN { [category["category"]]: category.categoryname }
 )
 """
+
+query_items_for_menu = """
+FOR category IN menu_categories
+    FILTER category.language == @language
+    FOR catfile in category.files
+        FOR file in files
+            FILTER file._key == catfile
+            SORT file.filenr
+            RETURN {displayName: file.displayName,
+                    textname: file.textname,
+                    filename: file.filename,
+                    category: file.category}
+"""
+
+query_items_for_filter_menu = """
+FOR category IN menu_categories
+    FILTER category.language == @language
+    SORT category.categorynr
+    FOR catfile in category.files
+        RETURN {filename: catfile}
```

Comment:
We're returning an array like this:
```
[
  {
    "filename": "dn1"
  },
  {
    "filename": "dn2"
  },
  {
    "filename": "dn3"
  },
]
```
but the `filename` key seems to be unnecessary here. If we just `RETURN catfile`, we'll get a simpler array.

Reply:
Yes, but the multi-combo-box needs an object with items with a certain name each. It is possible that way to also add more items like "displayname" that differ from the filename. Right now I only use the filename in the combo-box but maybe in the future that will change. Will have to see how it feels after a while.